src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -154,8 +171,32 @@ private static unsafe int SysReceive(SafeSocketHandle socket, SocketFlags flags,
             return checked((int)received);
         }
 
+        private static unsafe int SysWrite(SafeSocketHandle socket, ReadOnlySpan<byte> buffer, ref int offset, ref int count, out Interop.Error errno)
+        {
+            int sent;
+
+            fixed (byte* b = &MemoryMarshal.GetReference(buffer))
+            {
+                sent = Interop.Sys.Write(socket, &b[offset], buffer.Length);
```

Comment:
```suggestion
                sent = Interop.Sys.Write(socket, b + offset, buffer.Length);
```
?  I don't actually have a strong opinion, I just don't think we typically use the `&b[offset]` syntax in cases like this.  They'll result in exactly the same asm.  I _think_ `b + offset` will be slightly smaller IL.